CppCoreGuidelines.md

```diff
@@ -7940,7 +7940,37 @@ Casting to a reference expresses that you intend to end up with a valid object,
 
 ##### Example
 
-    ???
+        class Base {
+        public:
+            virtual void f() { }
+        };
+        
+        class Derived : public Base {
+        public:
+            void f() override { }
+        };
+        
+        class NotDerivedFromBase {
+        public:
+            virtual void f() { }
+        };
+        
+        class DerivedFromNotDerivedFromBaseAndFromBase : public NotDerivedFromBase, public Base {
```

Comment:
I don't think this name is too useful, as it is difficult to read,  an you immediately see what this class is derived from anyway.  Imho a simple Foo would be just fine.